samples/gpio-pci-idio-16.c

```diff
@@ -91,6 +91,15 @@ int main(int argc, char *argv[])
         err(EXIT_FAILURE, "missing MUSER device UUID");
     }
 
+    if (trans_sock) {
+        char *uuid = argv[optind];
+        for (int i = 0; i < strlen(uuid); i++) {
```

Comment:
you can use `strcspn` to delimit chunks without hypens, then move only those chunks instead of the whole string.